apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -58,6 +100,23 @@ router.get(
 router.post(
   '/:unsafe_lti13_course_instance_id',
   asyncHandler(async (req, res) => {
+    const instance = await queryRow(
+      sql.select_lti13_instance,
+      {
+        course_instance_id: res.locals.course_instance.id,
+        lti13_course_instance_id: req.params.unsafe_lti13_course_instance_id,
+      },
+      Lti13CombinedInstanceSchema,
+    );
+
+    const serverJobOptions = {
+      courseInstanceId: res.locals.course_instance.id,
+      userId: res.locals.user.user_id,
+      authnUserId: res.locals.authn_user.user_id,
+      type: 'lti13',
+      description: 'Some LTI operation',
```

Comment:
Consider making this more exact?

Reply:
This is the boilerplate that gets overwritten with more exact descriptions/details based on the __action.